requirements/dev-requirements.txt

```diff
@@ -2,118 +2,122 @@
-    # via -r dev-requirements.in
+    # via -r requirements/dev-requirements.in
 faker==26.0.0
     # via factory-boy
 filelock==3.13.1
     # via
-    #   -c requirements.txt
+    #   -c requirements/requirements.txt
     #   virtualenv
 freezegun==1.5.1
-    # via -r dev-requirements.in
+    # via -r requirements/dev-requirements.in
 h11==0.14.0
     # via
-    #   -c requirements.txt
+    #   -c requirements/requirements.txt
     #   httpcore
 httpcore==0.17.3
     # via
-    #   -c requirements.txt
+    #   -c requirements/requirements.txt
     #   httpx
 httpx==0.24.1
     # via
-    #   -c requirements.txt
+    #   -c requirements/requirements.txt
     #   pytest-httpx
 identify==2.6.0
     # via pre-commit
 idna==3.7
     # via
-    #   -c requirements.txt
+    #   -c requirements/requirements.txt
     #   anyio
     #   httpx
 iniconfig==2.0.0
     # via pytest
 invoke==2.2.0
-    # via -r dev-requirements.in
+    # via -r requirements/dev-requirements.in
 mock==5.1.0
-    # via -r dev-requirements.in
+    # via -r requirements/dev-requirements.in
 nodeenv==1.9.1
     # via pre-commit
 packaging==23.2
     # via
-    #   -c requirements.txt
+    #   -c requirements/requirements.txt
     #   build
     #   pytest
+pip==24.2
+    # via pip-tools
 pip-tools==7.4.1
-    # via -r dev-requirements.in
+    # via -r requirements/dev-requirements.in
 platformdirs==4.2.2
     # via virtualenv
 pluggy==1.5.0
     # via pytest
 pre-commit==3.7.1
-    # via -r dev-requirements.in
+    # via -r requirements/dev-requirements.in
 pyproject-hooks==1.1.0
     # via
     #   build
     #   pip-tools
 pytest==7.4.4
     # via
-    #   -r dev-requirements.in
+    #   -r requirements/dev-requirements.in
     #   pytest-cov
     #   pytest-django
     #   pytest-httpx
 pytest-cov==5.0.0
-    # via -r dev-requirements.in
+    # via -r requirements/dev-requirements.in
 pytest-django==4.8.0
-    # via -r dev-requirements.in
+    # via -r requirements/dev-requirements.in
 pytest-httpx==0.24.0
-    # via -r dev-requirements.in
+    # via -r requirements/dev-requirements.in
 python-dateutil==2.8.2
     # via
-    #   -c requirements.txt
+    #   -c requirements/requirements.txt
     #   faker
     #   freezegun
 pyyaml==6.0.1
     # via
-    #   -c requirements.txt
+    #   -c requirements/requirements.txt
     #   pre-commit
 ruff==0.7.0
-    # via -r dev-requirements.in
+    # via -r requirements/dev-requirements.in
+setuptools==75.2.0
+    # via pip-tools
 six==1.16.0
     # via
-    #   -c requirements.txt
+    #   -c requirements/requirements.txt
     #   python-dateutil
 sniffio==1.3.0
     # via
-    #   -c requirements.txt
+    #   -c requirements/requirements.txt
     #   anyio
     #   httpcore
     #   httpx
 termcolor==2.4.0
-    # via -r dev-requirements.in
+    # via -r requirements/dev-requirements.in
 virtualenv==20.26.3
     # via pre-commit
 watchfiles==0.22.0
-    # via -r dev-requirements.in
+    # via -r requirements/dev-requirements.in
```

Comment:
nit: it would be great to undo these comment changes since it makes it hard to see what's changed. 

I've updated the `requirements` task to prevent this in future: https://github.com/dimagi/open-chat-studio/commit/d1d8be2dec8a0b9dadb49bda06104c94cbb7ca5e

Reply:
https://github.com/dimagi/open-chat-studio/pull/781/commits/d951b0c1f9571eb6d3127820a5f8a955d1d5cb32